accepted/PSR-6-cache-meta.md

```diff
@@ -276,25 +276,59 @@ default control information is all that's possible), and because it's trivial
 for a particular implementation to include as an add-on should it desire to
 do so.
 
-5. People
+5. Errata
```

Comment:
Adding a section mid-document means re-flowing all remaining numbers. May I suggest that errata become a new section at the bottom of the document?

Reply:
Hrm. I had thought that's what we'd done elsewhere, but I went back and looked and the only PSR with Errata is PSR-2, which didn't have a metadoc in the first place so it's a bad example.